pkg/rpc/datacatalogservice/service.go

```diff
@@ -124,3 +132,74 @@ func NewDataCatalogService() *DataCatalogService {
 			catalogScope.NewSubScope("reservation")),
 	}
 }
+
+// Create and start the gRPC server
+func ServeInsecure(ctx context.Context, cfg *config.Config) error {
```

Comment:
where is the secure version?

Reply:
I didn't remove it. It looks like we don't have it.